plugins/package-managers/pub/src/main/kotlin/Pub.kt

```diff
@@ -609,7 +610,7 @@ class Pub(
 
                         RemoteArtifact(
                             url = "$hostUrl/packages/$rawName/versions/$version.tar.gz",
-                            hash = Hash.create(sha256)
+                            hash = Hash(sha256, HashAlgorithm.SHA256)
```

Comment:
After my related PR is merged, using constructor instead of `create()` would by-pass turning the `value` lowercase. At least it was my intention to make `create()` ensure the `value` passed to the constructor is `lowercase`. 

Maybe we can discuss it here in one place: https://github.com/oss-review-toolkit/ort/pull/8214#discussion_r1475838326

Reply:
So, from the linked discussion, I believe this could now just be changed to call the `create()` function, passing again, but pass the algorithm, right?